config-connector/solutions/iam/kpt/service-account/iampolicy.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicy
```

Comment:
Any particular reason to use IAMPolicy here?

Reply:
Discussed offline. Use iampolicymember instead now.